src/app/cart/pages/cart/cart.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit } from '@angular/core';
+
+import { CartItem } from '../../../core/models/cart-item';
+
+import { AppSettingService } from '../../../core/services/app-settings/app-setting.service';
+import { CartService } from '../../../core/services/cart/cart.service';
+
+@Component({
+  selector: 'app-cart',
+  templateUrl: './cart.component.html',
+  styleUrls: ['./cart.component.scss'],
```

Comment:
where is on push?
I askes to add it throw whole app previous time

Reply:
fixed